org.duckdns.bugfreeblog.mldonkeynext.json

```diff
@@ -0,0 +1,48 @@
+{
+  "app-id": "org.duckdns.bugfreeblog.mldonkeynext",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.9",
+  "sdk": "org.kde.Sdk",
+  "base": "io.qt.qtwebengine.BaseApp",
+  "base-version": "6.9",
+  "command": "mldonkey-next",
+  "finish-args": [
+    "--share=network",
+    "--socket=wayland",
+    "--device=dri",
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11"
+  ],
+  "modules": [
+    {
+      "name": "mldonkey-next",
+      "buildsystem": "simple",
+      "build-commands": [
+        "./create_qt_resource_files.sh",
+        "cmake -DCMAKE_INSTALL_PREFIX=/app .",
+        "cmake --build .",
+        "cmake --install .",
```

Comment:
this should be `buildsystem: cmake-ninja`

Reply:
1. build commands don't make sense with anything but `simple`
2. There are `shell` source if you really must. But this should be part of the build system.

Reply:
Updated.